tree-sitter-darklang/grammar.js

```diff
@@ -764,7 +775,7 @@ module.exports = grammar({
     apply: $ =>
       prec.right(
         seq(
-          field("fn", $.qualified_fn_name),
+          field("fn", choice($.qualified_fn_name, $.fn_identifier)),
```

Comment:
because I updated `qualified_fn_name` to require one module name I had to add fn_identifier here, not sure about this decision.

Reason behind the `qualified_fn_name`  update:
`1 |> x` was parsed as `pipe_fn_call(qualified_fn_name) ` instead of `pipe_variable_or_fn_call`

Reply:
I'm very hesitant about updating `qualified_fn_name` to require at least one module name - we should avoid that.

> `1 |> x was parsed as pipe_fn_call(qualified_fn_name)`  instead of `pipe_variable_or_fn_call`

Hm at first glance, that seems like the right thing to do, actually?

Ah, I guess the counterargument would be when you allow for inline lambdas / fn declarations, like:
```fsharp
let myLambda = fun x -> x + 1
1 |> myLambda |> etc
```

I suppose, in that case, `myLambda` is more of a variable identifier than a fn name.

So, I think
- WT should be one of those "maybe it's a variable name, maybe it's a fn name" things
- WT2PT should _try_ to find a fn of the name -- if it can't find one, it's an `EPipeVariable` - if it can, it's an `EPipeVariableOrFnCall` - or something like that

hope that makes sense